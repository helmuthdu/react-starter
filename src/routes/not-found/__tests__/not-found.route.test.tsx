import { render } from '@testing-library/react';
import * as React from 'react';
import NotFoundRoute from '../not-found.route';

describe('Route -> NotFound', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<NotFoundRoute />);
    expect(asFragment()).toMatchSnapshot();
  });
});
