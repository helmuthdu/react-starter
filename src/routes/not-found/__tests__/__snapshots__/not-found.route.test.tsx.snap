// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Route -> NotFound > should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      You are here!
    </h1>
    <h2>
      But nothing found for you #404
    </h2>
  </div>
</DocumentFragment>
`;
